tests/components/plex/conftest.py

```diff
@@ -401,12 +415,14 @@ def mock_plex_calls(
     requests_mock.get(f"{url}/library/metadata/300/allLeaves", text=grandchildren_300)
 
     requests_mock.get(f"{url}/library/metadata/1", text=media_1)
+    requests_mock.get(f"{url}/library/metadata/20", text=media_20)
     requests_mock.get(f"{url}/library/metadata/30", text=media_30)
     requests_mock.get(f"{url}/library/metadata/100", text=media_100)
     requests_mock.get(f"{url}/library/metadata/200", text=media_200)
 
     requests_mock.get(f"{url}/library/metadata/20/children", text=children_20)
     requests_mock.get(f"{url}/library/metadata/30/children", text=children_30)
+    requests_mock.get(f"{url}/library/metadata/30/allLeaves", text=children_20)
```

Comment:
The query asks for all episodes in the show, but the selected fixture is all episodes in the season; this is intentional. Since the show currently only has 1 season, I figured I could simply re-use its fixture with the list of episodes.

Please let me know if this is not desirable and I will duplicate it into a separate fixture instead.

Reply:
I think that's reasonable.